### PR TITLE
Cleaning up a handful of typos

### PR DESCRIPTION
**Pull request by justingonder**

### CODE REVIEW

**Comment by Bbkctp** [comment]
Can one of the admins verify this patch?

**Comment by ajrbyers** [comment]
Test this please.

File: src/copyediting/migrations/0001_initial.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('assigned', models.DateTimeField(default=django.utils.timezone.now)),
                 ('notified', models.BooleanField(default=False)),
-                ('decision', models.CharField(blank=True, choices=[('accept', 'Accept'), ('corrections', 'Corrections Required')], help_text="Select a decision that reflects your actions. If you've uploaded a newversion of the file, select Corrects Required, if you're happy with thecopyedit, select Accept.", max_length=20, null=True, verbose_name='Your Decision')),
+                ('decision', models.CharField(blank=True, choices=[('accept', 'Accept'), ('corrections', 'Corrections Required')], help_text="Select a decision that reflects your actions. If you've uploaded a new version of the file, select Corrections Required, if you're happy with the copyedit, select Accept.", max_length=20, null=True, verbose_name='Your Decision')),
```
Comment (mauromsl) [inline src/copyediting/migrations/0001_initial.py]: This changes the original database migration but not the model itself, which is used at run time. The same change needs to happen in `janeway/src/copyediting/models.py` lines 111-113.

**Comment by justingonder** [comment]
@mauromsl thanks! i'm still learning :) have updated the other file now as well.

**Pull request merged**